### PR TITLE
[QUAD-260] Removed JAXB annotations from DTOs.

### PR DESCRIPTION
**Pull request by chiraag-subramanian**

### CODE REVIEW

**Review by abhikrish503501 — APPROVED** [review]
The annotations have been removed properly and indentation is good

**Review by Mihika0409 — APPROVED** [review]

**Comment by jdamerow** [comment]
ok to test

File: Quadriga/src/main/java/edu/asu/spring/quadriga/dto/WorkspaceDTO.java [hunk 1]
```diff
@@ -21,7 +21,6 @@
 import javax.persistence.OneToMany;
 import javax.persistence.OneToOne;
 import javax.persistence.Table;
-import javax.xml.bind.annotation.XmlTransient;
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/dto/WorkspaceDTO.java:24]: don't delete just the imports; you also need to delete where they are used. eclipse should have given you an error if you do that.

**Review by jdamerow — CHANGES_REQUESTED** [review]
make sure you don't break the code; eclipse should have been all over with compiler errors;

**Review by abhikrish503501 — APPROVED** [review]
Removed the annotations

**Review by jdamerow — APPROVED** [review]

**Pull request merged**